vuu-ui/packages/vuu-data/src/server-proxy/array-backed-moving-window.ts

```diff
@@ -41,6 +45,12 @@ export class ArrayBackedMovingWindow {
   }
 
   setRowCount = (rowCount: number) => {
+    if (
```

Comment:
there are a couple of issues here. Firstly, why incur the cost of a function call when the function is just returning a static value - just reference   loggingSettings.loggingLevel directly. Worse, we make 2 function calls.

Supposing you define the loglevels like this:
```
const LogLevel = {
 low: 1,
medium: 2,
high: 3
}
```

then , instead of having to write code like
```
if (logLevel === 'high' || logLevel === 'medium')
```

you can just use 
``
if (logLevel >= logLevel.medium)
``